plugins/llvm/llvm_binary.hpp

```diff
@@ -18,93 +18,93 @@
 extern "C" void llvm_binary_fail(const char*) LLVM_ATTRIBUTE_NORETURN ;
 
 LLVM_ATTRIBUTE_NORETURN void llvm_binary_fail (const llvm::error_code& ec) {
-    llvm_binary_fail(ec.message().c_str());
+  llvm_binary_fail(ec.message().c_str());
 }
 
 namespace llvm { namespace object {
 
-template <typename T>
-content_iterator<T>& operator++(content_iterator<T>& a) {
-    error_code ec;
-    a.increment(ec);
-    if(ec) llvm_binary_fail(ec);
-    return a;
-}
+    template <typename T>
```

Comment:
We do not indent inside namespaces. And use 4 whitespaces for indentation. Add the following to yours emacs config, if you want your emacs to produce correct indentation:
```lisp
(setq-default c-basic-offset 4)
(add-hook 'c++-mode-hook
          '(lambda ()
             (require 'auto-complete-c-headers)
             (add-to-list 'ac-sources 'ac-source-c-headers)
             (c-set-style "stroustrup")
             (c-toggle-hungry-state)
             (c-set-offset 'innamespace 0)))
```